examples/server/distributed/main.go

```diff
@@ -144,15 +144,19 @@ func parent() {
 }
 
 func child(port string) {
-	// Create a server with a single tool that increments a counter.
-	server := mcp.NewServer(&mcp.Implementation{Name: "counter"}, nil)
+	// Create a server with a single tool that increments a counter and sends a notification.
+	opts := &mcp.ServerOptions{
```

Comment:
I'd rather not introduce a new exported symbol. Let's hardcode the default level.
And let's make it Info, not Debug. Otherwise servers are likely to spam clients who don't care.
(In my experience, debug logs are often 10x the size of info logs.)

Reply:
Alright, changed it.